content/css/sel4.css

```diff
@@ -286,3 +286,61 @@ a.divLink {
   font-size:20px;
   font-weight: bold;
 }
+
+.news {
+  width: auto;
+  height:auto;
+  padding-top:5px;
+  padding-left:10px;
+  padding-right:5px;
+  margin-top:20px;
+  margin-left:60px;
+  margin-right:60px;
+  border-radius: 10px;
+  background-color: #e5e9ee; /* ligher than hover D6DBE4 */
+}
+
+.news:hover {
+  background-color: #D6DBE4; /* neutralLighter */
+}
+
+.news-column {
+  float: left;
+  padding: 5px;
+}
+
+.news-clearfix::after {
+  content: "";
+  clear: both;
+  display: table;
+}
+
+.news-title {
+  width: 75px;
+  font-size: 200%;
+  font-family: courier;
+  line-height: 80%;
+}
+
+.news-content {
+  width: auto;
+  font-family: courier;
+}
+
+
+.news-item {
+  width: auto;
+  height:auto;
+  padding:5px;
+  margin-top:20px;
+  margin-bottom:5px;
+  border-radius: 5px;
+  background-color: #DCDCDC;
```

Comment:
This should be a colour from the colour scheme in https://github.com/seL4/branding/blob/master/colour-scheme/foundation-colours.scss

Reply:
changed. thanks